procedures.sql

```diff
@@ -0,0 +1,1136 @@
+CREATE PROCEDURE AddStudent
+    @first_name VARCHAR(50),
+    @last_name VARCHAR(50),
+    @address VARCHAR(200),
+    @email VARCHAR(50),
+    @phone VARCHAR(20)
+AS
+BEGIN
+    INSERT INTO Student (name, surname, address, email, phone_number)
```

Comment:
there is a unconsistency between `name - surname` and `first_name - last_name`. Pick one of the pairs and apply everywhere.

Reply:
first_name, last_name seems clearer to me